ci-operator/step-registry/aks/provision/aks-provision-commands.sh

```diff
@@ -6,22 +6,30 @@ AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
 AZURE_AUTH_CLIENT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientId)"
 AZURE_AUTH_CLIENT_SECRET="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientSecret)"
 AZURE_AUTH_TENANT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .tenantId)"
+AZURE_LOCATION="$LEASED_RESOURCE"
 
-CLUSTER="${NAMESPACE}-${UNIQUE_HASH}"
-RESOURCEGROUP="$(<"${SHARED_DIR}/resourcegroup")"
+RESOURCE_NAME_PREFIX="${NAMESPACE}-${UNIQUE_HASH}"
 
 az --version
 az login --service-principal -u "${AZURE_AUTH_CLIENT_ID}" -p "${AZURE_AUTH_CLIENT_SECRET}" --tenant "${AZURE_AUTH_TENANT_ID}" --output none
 
+set -x
```

Comment:
Shall we remove this line, too ?

Reply:
I wanted to keep it but okay I will remove it after the rehearsals terminate.

Reply:
Removed.